grafana-plugin/src/components/GBlock/Block.styles.ts

```diff
@@ -16,7 +16,7 @@ export const getBlockStyles = (theme: GrafanaTheme2) => {
       }
 
       &--hover:hover {
-        background: var(--hover-selected);
+        background: ${theme.isLight ? '#f4f5f5' : 'rgba(204, 204, 220, 0.12)'};
```

Comment:
Shouldn't we use here some vars from theme?

Reply:
This one is a bit odd. If you look for #f4f5f5 in core grafana it will point to `theme.colors.background.canvas` for light, while for dark the other one points to `theme.colors.action.selected` which seems a bit strange if you read it like

`${theme.isLight ? theme.colors.background.canvas : theme.colors.action.selected};`

But I can easily change it, though I must admit it looks a bit odd to either pick canvas background or the color for a selected action 😆

Reply:
I see what you mean. I just want to avoid using raw values directly. Then I don't have any preference whether to add them to Colors enum in utils.styles or use this strange values from theme